blog/sudoko_blog_structure.md

```diff
@@ -249,14 +249,20 @@ How accurate? Compare to different implementations both AI and programmatic ones
 
 We have learnt a great deal throughout this journey to solving sudoku puzzles with neural networks. We discovered how different architectures are suited to different tasks, how to apply improvement techniques to poorly performing base models, and how to work with datasets to train models for the task at hand.
 
-We have also learnt (confirmed?) that neural networks are not the best way to solve sudoku puzzles. There are numerous drawbacks including:
+We have also learnt (confirmed?) that neural networks are not the best tools to solve sudoku puzzles. There are drawbacks aplenty including:
 
 - Poor accuracy, especially on puzzles with a great number of blanks
-- Solving different size puzzles requires re-training
-- Model files take up significant storage
+- Solving different size puzzles (for example 4x4 or 12x9) requires architecture tweaks and retraining
+- The model parameters take up more storage than the lines of code of a written solver do (though the models are very small in the realm of neural networks)
 
 On top of all this, algorithmic solvers already exist which have 100% accuracy - and they can often tell when a sudoku has no solution, something which our AI has no concept of. Additionally, these programmatic solvers are able to determine multiple solutions to sudokus which do not have unique solutions - our AI is not able to output multiple solutions to the same puzzle, it is deterministic once trained.
 
-Despite its many shortcomings, one thing we should not expect of an AI sudoku solver is that it be infallible. It may be thought of not as code but as a small brain, distilled for the sole purpose of solving sudoku puzzles. People make mistakes on sudokus all the time - being tough on this AI, which figures out possible solutions to millions of puzzles in seconds, would be unfair.
+There are also many positive aspects of solving sudoku puzzles with neural networks such as:
+
+- No need to write code which solves by constraints or deduction, we built and trained an initial model within half an hour
+- Puzzles need not be logically soluble, the AI solver may find solutions to sudokus which deductive programmatic solvers could never work out
```

Comment:
Not sure the dissolving properties of the puzzle has much of an impact on the performance of the programmatic solvers ;)

```suggestion
- Puzzles need not be logically solvable, the AI solver may find solutions to sudokus which deductive programmatic solvers could never work out
```